plugins/src/webrtcsink/imp.rs

```diff
@@ -582,7 +611,7 @@ impl VideoEncoder {
     pub fn set_bitrate(&mut self, element: &super::WebRTCSink, bitrate: i32) {
         match self.factory_name.as_str() {
             "vp8enc" | "vp9enc" => self.element.set_property("target-bitrate", bitrate),
-            "x264enc" | "nvh264enc" | "vaapih264enc" | "vaapivp8enc" => self
+            "x264enc" | "nvh264enc" | "vaapih264enc" | "vaapivp8enc" | "nvv4l2h264enc" | "nvv4l2vp8enc" => self
```

Comment:
So the `bitrate` property in those two elements is in kilobits per second? All the examples I can find use fairly high values, and https://docs.nvidia.com/metropolis/deepstream/dev-guide/text/DS_plugin_gst-nvvideo4linux2.html seems to indicate that the value is in bits per second actually. Have you actually tested this patch?

Reply:
Yes you are right it is in bits per second that was my mistake. Yes it is tested, with that bitrate everything works, simply the video quality will be bad and with artifacts. With a bitrate of 4M or above the quality of the video will be good and no artifacts. I will make the necessary changes.

Reply:
Ok, thanks, have you tested congestion control with the elements btw?

Reply:
Yes, it was tested with congestion control enabled.